lisp/forge-topic.el

```diff
@@ -205,8 +205,7 @@ This variable has to be customized before `forge' is loaded."
 (defun forge--topic-string-to-number (s)
   (save-match-data
     (if (string-match "\\`\\([!#]\\)?\\([0-9]+\\)" s)
-        (* (if (equal (match-string 1 s) "!") -1 1)
-           (string-to-number (match-string 2 s)))
+        (string-to-number (match-string 2 s))
```

Comment:
Is this a rebase accident?

Reply:
well no, I don't really understand why we need to return a negative value when it match "!" (gitlab PR).

If we want to pull single topic, with a negative value it failed.